code/modules/research/machinery/protolathe.dm

```diff
@@ -32,3 +32,38 @@
 /obj/machinery/rnd/production/protolathe/offstation
 	offstation_security_levels = FALSE
 	circuit = /obj/item/circuitboard/machine/protolathe/offstation
+
+/obj/machinery/rnd/production/protolathe/offstation/ghost_cafe
+	categories = list(
+								"Power Designs",
+								"Medical Designs",
+								"Bluespace Designs",
+								"Stock Parts",
+								"Equipment",
+								"Tool Designs",
+								"Mining Designs",
+								"Electronics",
+								"Weapons",
+								"Ammo",
+								"Firing Pins",
+								"Computer Parts",
+								"Exosuit Modules",
+								"Hydroponics Machinery",
+								"Misc. Machinery",
+								"Computer Parts"
+								)
+	console_link = FALSE
+	requires_console = FALSE
+	allowed_buildtypes = PROTOLATHE | IMPRINTER
+/obj/machinery/rnd/production/protolathe/offstation/ghost_cafe/Initialize(mapload)
+	. = ..()
+	host_research = SSresearch.admin_tech
+
+/obj/effect/landmark/ghost_protolathe
+	name = "Ghost Cafe Protolathe"
+	icon = 'icons/obj/machines/research.dmi'
+
+/obj/effect/landmark/ghost_protolathe/Initialize()
```

Comment:
adding a landmark to spawn a protolathe instead of just placing one seems ??? to me